modules/cli-options/src/main/scala/scala/cli/commands/PackageOptions.scala

```diff
@@ -45,6 +45,10 @@ final case class PackageOptions(
     preamble: Boolean = true,
   @Group("Package")
   @Hidden
+  @HelpMessage("For assembly JAR, whether to specify a main class in the JAR manifest")
+    mainClassInManifest: Option[Boolean] = None,
+  @Group("Package")
+  @Hidden
```

Comment:
Why make it hidden, though?

Reply:
Because it feels kind of arcane to me… I could un-hide though.

Reply:
IMO hiding it only makes it more arcane, while it might be useful to users (even if obscure until we cover it in a cookbook

Reply:
I think we should only hide the stuff that's too unstable/experimental for users to rely on or it's something targeting the Scala CLI devs

Reply:
I think we hide much more stuff than this. Many options are hidden so that we don't overflow users when they pass `--help`.

Reply:
we could consider specialised `--help` sections in a few spots, I suppose.

Reply:
I'd still un-hide it, otherwise it won't become useful to anyone, I think.
but yeah, your points are fair as well, so I'm leaving the decision to you.

Reply:
Merging so that I don't have to wait for the CI again. We can always revisit that later, or when adding specialised help options.